### ISSUE TITLE
[BITV] 9.2.4.7/6.2 - Selected list items in combo boxes receive a poorly recognizable keyboard focus. Since the focus highlighting is solely accomplished through a color change, the contrast ratio between the focused and unfocused state should be at least 3:1. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37015]
- [x] Fix
- [x] Backport

Personal info - Personal settings - Nextcloud
https://try.nextcloud.com/ltd/a11y/index.php/settings/user
<img width="396" alt="09dbe8e4c16bc629e14fecd05fa10016" src="https://user-images.githubusercontent.com/1315170/222543184-ef35960e-a1eb-4e68-9fa7-96786a84ddec.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v6-n2

</details>

- [x] 9.2.4.7/8.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v8-n2
